Jenkinsfile

```diff
@@ -24,11 +26,27 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             coreos-assembler fetch
             """)
         }
+
+        def prevBuildID = null
+        if (utils.shwrap_rc("test -f builds/latest")) {
```

Comment:
I'll note that there's a `fileExists` step I tried to use here, but it sadly doesn't work correctly inside the Kubernetes container. So we have to resort to just `test -f`.